src/test/java/com/salesforce/dev/EditProduct.java

```diff
@@ -0,0 +1,68 @@
+package com.salesforce.dev;
+import com.salesforce.dev.pages.Base.NavigationBar;
+import com.salesforce.dev.pages.Product.ProductBuilder;
+import com.salesforce.dev.pages.Product.ProductDetails;
+import com.salesforce.dev.pages.Product.ProductForm;
+import com.salesforce.dev.pages.Product.ProductsHome;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import com.salesforce.dev.framework.Environment;
+import com.salesforce.dev.pages.*;
+/**
+ * Created by Monica Pardo on 6/10/2015.
+ */
+
+
+
+public class EditProduct {
+
+    private LoginPage loginPage;
+    private ProductsHome productsHome;
+    private ProductDetails productDetails;
+    private ProductForm productForm;
+    private HomePage homePage;
+    private MainPage mainPage;
+    private String productName="New product";
+    private String prodCode="Codigo1";
+    private String prodDesc="this is a new product";
+    private String productNewName="New product update";
+    private String prodNewCode="Codigo2";
+    private String prodNewDesc="this is a new product update";
+
+
+    @BeforeMethod
+    public void setUp() {
+        homePage = new HomePage();
+        String userNameValue=Environment.getInstance().getPrimaryUser();
+        String passwordValue= Environment.getInstance().getPrimaryPassword();
+        mainPage = homePage.loginAs(userNameValue, passwordValue);
```

Comment:
Create a loginAsPrimaryUser method in order to avoid using the userName and password